core/src/main/java/org/apache/accumulo/core/client/admin/SummaryRetriever.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.accumulo.core.client.admin;
+
+import java.util.Map;
+
+import org.apache.hadoop.io.Text;
+
+public interface SummaryRetriever {
+
+  /**
+   * Allows optionally setting start row before retrieving data.
+   */
+  SummaryRetriever start(Text startRow);
```

Comment:
My assumption was that the `startRow` would essentially refer to the first Tablet to read for summaries. Since the summaries are at the file-level, you would get a summarization "greater" than what you actually specified (would be good to make sure is clear via documentation).